firmware/Core/Inc/gps/gps_pps_interrupt.h

```diff
@@ -0,0 +1,17 @@
+
+#ifndef __INCLUDE_GUARD__GPS_PPS_INTERRUPT_H
+#define __INCLUDE_GUARD__GPS_PPS_INTERRUPT_H
+
+#include "stm32l4xx_hal.h"
+
+extern uint32_t pps_counter;
+
+extern uint8_t gps_pps_enabled;
```

Comment:
Please prefix variables with `GPS_` (see `docs/` -> Namespace Guide for more details).

Please rename to `GPS_pps_interrupt_enabled`